hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/map/SqlCompactTest.java

```diff
@@ -390,49 +360,30 @@ public void test_allTypes() throws IOException {
                 + ")"
         );
 
-        sqlService.execute("SINK INTO " + to + " SELECT 13, 'a', f.* " + " FROM " + from + " f");
+        sqlService.execute("SINK INTO " + to + " (id, \"character\", string, \"boolean\", byte, short, \"int\", long,"
```

Comment:
You can still use all-types-connector - just select all the fields except `object`.